mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -480,6 +481,13 @@ is set to true. Defaults to ZONAL.`,
 											},
 										},
 									},
+                                                                        "ssl_mode": {
+                                                                                Type:         schema.TypeString,
+                                                                                Optional:     true,
+                                                                                ValidateFunc: validation.StringInSlice([]string{"SSL_MODE_UNSPECIFIED", "ALLOW_UNENCRYPTED_AND_ENCRYPTED", "ENCRYPTED_ONLY", "TRUSTED_CLIENT_CERTIFICATE_REQUIRED"}, false),
```

Comment:
Nit: You probably don't need `SSL_MODE_UNSPECIFIED` (the user can just not specify the field), unless there is some specific use for it.

Reply:
SG. The value has been removed